app/services/identity_reconciler_test.go

```diff
@@ -84,7 +84,7 @@ func TestIdentityReconciler(t *testing.T) {
 		assert.Nil(t, found)
 	})
 
-	t.Run("update missing email after oauth migration table", func(t *testing.T) {
+	t.Run("update changed email after oauth migration table", func(t *testing.T) {
```

Comment:
This is still testing "update missing email" - I meant should add a test if you added support for the email change discussed above.